### DISCUSSION TITLE
Collapsible settings sections

### DISCUSSION THREAD

**Discussion by MightyHelper** [ppy/osu#13089] [Ideas]
**Describe the new feature:** 
Fold a block.
On click of the arrow, the arrow rotates 90deg and expands/collapses a block in settings.
**Proposal designs of the feature:** 
![OsuFoldSuggestion1](https://user-images.githubusercontent.com/32612290/69680739-8ddfb100-108a-11ea-92df-905d0ae5cdb1.png)
Of course here the block would contract only leaving the name and arrow.
![OsuFoldSuggestion2](https://user-images.githubusercontent.com/32612290/69680740-8ddfb100-108a-11ea-9952-4a57d61a02f7.png)
Made the arrow blue so easy to see, could change colour.
In fact, it probably could be new skin element
![OsuFoldSuggestionArrow](https://user-images.githubusercontent.com/32612290/69680884-12caca80-108b-11ea-8770-153b04b17731.png)
^^ Here is the simple arrow skin I came up with in a few seconds.

**Comment by peppy** [comment]
Definitely would be useful to have going forward. The arrow shouldn't be blue though.

**Comment by MightyHelper** [comment]
Yeah, I made it blue just so it was visible, maybe a #d192ae?
![image](https://user-images.githubusercontent.com/32612290/69747383-0dba5980-1125-11ea-8bda-102076ea0b08.png)
or stick with the classic #ff66aa
![image](https://user-images.githubusercontent.com/32612290/69747570-5e31b700-1125-11ea-92a1-2d5e63dc3318.png)
(White seems a bit boring)

**Comment by EVAST9919** [comment]
Should it be added as a separate UI element or it will be used only in settings?

**Comment by peppy** [comment]
Depends on how the implementation looks.
